datalake/src/main/java/com/sequenceiq/datalake/controller/sdx/SdxController.java

```diff
@@ -502,6 +505,12 @@ public void submitDatalakeDataSizesInternal(@TenantAwareParam String crn, String
         sdxBackupRestoreService.submitDatalakeDataInfo(operationId, dataSizesJson, ThreadBasedUserCrnProvider.getUserCrn());
     }
 
+    @Override
+    @CheckPermissionByResourceName(action = AuthorizationResourceAction.DATALAKE_VERTICAL_SCALING)
```

Comment:
If we use the same action as vertical, should we change this name to avoid misunderstanding?

Reply:
Please define a new right for vertical scaling.